core/src/main/java/tachyon/UnderFileSystemHdfs.java

```diff
@@ -293,8 +294,14 @@ public boolean isFile(String path) throws IOException {
       String[] rtn = new String[files.length];
       int i = 0;
       for (FileStatus status : files) {
+        TachyonURI filePathURI = new TachyonURI(status.getPath().toUri().toString());
+        String filePath =  NetworkUtils.replaceHostName(filePathURI).toString();
+        if (filePath.length() < path.length()) {
+          LOG.warn("File Path should not be shorter than base dir path.  File Path: "
+              + filePath + " and Base Dir Path: " + path);
+        }
```

Comment:
Is this check still necessary? Since we will get indexoutofbounds shortly after.